app/controllers/progress_job/progress_controller.rb

```diff
@@ -2,10 +2,14 @@ module ProgressJob
   class ProgressController < ActionController::Base
 
     def show
-      @delayed_job = Delayed::Job.find(params[:job_id])
-      percentage = !@delayed_job.progress_max.zero? ? @delayed_job.progress_current / @delayed_job.progress_max.to_f * 100 : 0
-      render json: @delayed_job.attributes.merge!(percentage: percentage).to_json
+      @delayed_job = Delayed::Job.find_by(id: params[:job_id])
```

Comment:
I ended up writing something custom for my application.  Let me extract what  did and update this PR